cns/restserver/api.go

```diff
@@ -1245,14 +1247,21 @@ func (service *HTTPRestService) publishNetworkContainer(w http.ResponseWriter, r
 		returnCode = types.UnsupportedVerb
 	}
 
+	// this is an ugly hack because DNC depends on checking this status code,
+	// even though it's no longer necessary for it to do so. It does not need to
+	// handle retries because retries will be handled by the nmagent client in
+	// CNS. However, there are versions of DNC out there that still rely on this
+	// body being present.
+	publishResponseBody := fmt.Sprintf(`{"httpStatusCode":"%d"}`, publishStatusCode)
```

Comment:
It seems DNC also expects a similar response on the unpublish path

Reply:
Fixed in b4443fd